src/Http/Controllers/Configuration/SsoController.php

```diff
@@ -70,10 +112,26 @@ public function getEnableAll()
      * @return \Illuminate\Http\RedirectResponse
      * @throws \Seat\Services\Exceptions\SettingException
      */
-    public function getRemoveAll()
+    public function getDeleteProfile(int $id)
     {
 
-        setting(['sso_scopes', []], true);
+        $scopes = collect(setting('sso_scopes', true));
+
+        // default profile cannot be removed
+        $default_profile = $scopes->first(function ($item) {
+            return $item->name == 'default';
```

Comment:
You add a `default` property to your object, why not using it instead the name ?
I think it would be more reliant ;)